src/components/recipeList.tsx

```diff
@@ -1,52 +1,60 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Recipe from "./recipe";
+import RecipeForm from "./recipeform";
+import { IRecipeItem } from "../types/data";
 import "./recipeList.css";
 
 function RecipeList() {
-  const recipes: {
-    id: number;
-    recipeName: string;
-    description: string;
-    servings: number;
-    prepTime: string;
-    cookTime: string;
-    sproutyPie: boolean;
-    lollysPantry: boolean;
-  }[] = [
-    {
-      id: 0,
-      recipeName: "Cole's Roasted Salsa",
-      description:
-        "A delicious spicy salsa made with roasted tomatoes and serrano peppers",
-      servings: 10,
-      prepTime: "20 minutes",
-      cookTime: "10 minutes",
-      sproutyPie: false,
-      lollysPantry: true,
-    },
-    {
-      id: 1,
-      recipeName: "Tofu Rice Bowls",
-      description:
-        "Air fried tofu sits on a bed of jasmine rice with teriyaki sauce, sesame seeds and green onion toppings.",
-      servings: 4,
-      prepTime: "10 minutes",
-      cookTime: "15 minutes",
-      sproutyPie: true,
-      lollysPantry: false,
-    },
-  ];
+  const [recipes, setRecipes] = useState<IRecipeItem[]>([]);
+  const [isUpdate, setUpdate] = useState<boolean>(false);
+
+  const getRecipes = async () => {
```

Comment:
It would be nice to move these API calls into a separate file/module, to achieve separation of concerns and enable testing the `RecipeList` component in isolation.

Reply:
Also, this would make it easy to refactor every API call to use a different base URL. Once deployed, we can't use localhost:3000.